addons/account_edi_ubl_cii/models/account_edi_xml_ubl_bis3.py

```diff
@@ -138,6 +138,8 @@ def _get_partner_party_vals(self, partner, role):
             })
         if partner.country_id.code == "LU" and 'l10n_lu_peppol_identifier' in partner._fields and partner.l10n_lu_peppol_identifier:
             vals['endpoint_id'] = partner.l10n_lu_peppol_identifier
+        if partner.country_id.code == "SE" and partner.vat:
+            vals['endpoint_id'] = partner.vat.replace("SE", "")[:-2]
```

Comment:
Don't forget to remove this in 16.2, as the `peppol_endpoint` field exists (and can contain a value different from the vat).